f2/src/forms/ConfirmationForm.js

```diff
@@ -42,30 +42,31 @@ export const ConfirmationForm = (props) => {
           <Trans id="analystReport.selfHarmWord" />
         </div>
       ) : null}
-      <Form
-        onSubmit={props.onSubmit}
-        render={({ handleSubmit }) => (
-          <Stack
-            as="form"
-            onSubmit={handleSubmit}
-            shouldWrapChildren
-            spacing={6}
-          >
+      <Formik
+        initialValues={{}}
+        onSubmit={(values) => {
+          props.onSubmit(values)
```

Comment:
Since this form is used to confirm the report, and there are no values present there is no reason to pass values as a param to onSubmit. See ConfirmationPage.js, there is no param for the onSubmit prop.

Reply:
Good catch! I changed it. Thanks!